var/spack/repos/builtin/packages/py-plotnine/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.directives import depends_on, version
+
+
+class PyPlotnine(PythonPackage):
+    """plotnine is an implementation of a grammar of graphics in Python, it is
+    based on ggplot2. The grammar allows users to compose plots by explicitly
+    mapping data to the visual objects that make up the plot."""
+
+    pypi = "plotnine/plotnine-0.8.0.tar.gz"
+
+    version(
+        "0.8.0",
+        sha256="39de59edcc28106761b65238647d0b1f6212ea7f3a78f8be0b846616db969276",
+    )
+
```

Comment:
```python
depends_on('python@3.6:', type=('build', 'run'))
```

Reply:
Done in 41dd0358424c3cd75acba4ee5226381a22fb14d3.